tests/tests_unit/test_data_classes/test_contextualization.py

```diff
@@ -30,3 +44,108 @@ def test_wait_for_completion_running(self, job):
     def test_wait_for_completion_completed(self, job):
         job.wait_for_completion(timeout=1)
         assert job.status == "Completed"
+
+
+class TestJobBundle:
+    def test_DetectJobBundle_completed(self, mock_base_job_response) -> None:
+        _job_ids = [1, 2]
+        completed_job_mock = mock_base_job_response
+        with monkeypatch_cognite_client() as mock_client:
+            requestMock = Mock(
+                json=lambda: {
+                    "items": [
+                        completed_job_mock,
+                    ]
+                }
+            )
+            mock_client.diagrams._get.return_value = Mock(
+                json=lambda: {
+                    "createdTime": 1337,
+                    "items": [{"annotations": [], "fileId": 101}],
+                    "jobId": 1,
+                    "status": "Completed",
+                    "statusCount": {"completed": 1},
+                    "statusTime": 1337,
+                }
+            )
+            mock_client.diagrams._post.return_value = requestMock
+
+            job_bundle = DetectJobBundle(cognite_client=mock_client, job_ids=_job_ids)
+            assert job_bundle.job_ids == _job_ids
+            s, f = job_bundle.result
+            assert len(s) == 2
+            assert len(f) == 0
+            assert s[0]["status"] == "Completed"
+            assert s[1]["status"] == "Completed"
+
+    @patch("cognite.client.data_classes.contextualization.DetectJobBundle._WAIT_TIME", 0)
+    def test_DetectJobBundle_one_running(self, mock_base_job_response) -> None:
+        completed_job_mock = mock_base_job_response
+        running_job_mock = {**mock_base_job_response, **{"status": "Running", "jobId": 2}}
+        running_job_completed_mock = {**mock_base_job_response, **{"status": "Completed", "jobId": 2}}
+        with monkeypatch_cognite_client() as mock_client:
+            requestMock = Mock(json=lambda: {"items": [completed_job_mock, running_job_mock]})
```

Comment:
Could you also add a test that makes sure that no two JobBundles can be "active" at the same time?

Reply:
done